android/sdk/src/main/java/org/jitsi/meet/sdk/PictureInPictureModule.java

```diff
@@ -18,10 +18,53 @@ static boolean isPictureInPictureSupported() {
         return Build.VERSION.SDK_INT >= Build.VERSION_CODES.O;
     }
 
+    /**
+     * A hint which indicates whether the app should go to the picture in picture mode when user
+     * minimizes it.
+     */
+    private boolean wantsToBeInPiPMode = false;
```

Comment:
This flag would probably fit better into the JitsiMeetView instance, but I don't know how to make it happen. I don't know how to access JitsiMeetView instance from the JS side. It seems impossible at the moment.